dbt_coves/templates/source_model_props.yml

```diff
@@ -14,10 +14,9 @@ models:
   - name: {{ model.lower() }}
     columns:
 {%- for cols in nested.values() %}
-  {%- for col in cols %}
-      - name: {{ col.lower().replace(" ","_").replace(":","_").replace("(","_").replace(")","_") }}
-  {%- endfor %}
+      - name: {{ cols["key"].lower().replace(" ","_").replace(":","_").replace("(","_").replace(")","_") }}
+      - description: {{ cols["description"].lower().replace(" ","_").replace(":","_").replace("(","_").replace(")","_") }}
```

Comment:
@LeonardoPelanda  replacing in this way is not needed or the description will be altered:
"Country Code" will become "country_code_". Description does not need replacing initially. Maybe some sanitizing in the future.